src/pages/Environments.js

```diff
@@ -196,7 +195,7 @@ const Environments = () => {
   const totalRuntimeCost = _.sum(_.map(runtimeCost, runtimes))
   const totalAppCost = _.sum(_.map(getGalaxyComputeCost, apps))
   const totalCost = totalRuntimeCost + totalAppCost
-  const totalDiskCost = _.sum(_.map(getPersistentDiskCostMonthly, disks))
+  const totalDiskCost = _.sum(_.map(disk => getPersistentDiskCostMonthly(disk, getRegionFromZone(disk.zone)), disks))
```

Comment:
Out of scope for this PR - but the `getPersistentDiskCostMonthly` and `getRegionFromZone` is triggering a code smell for me since both of these functions are working off of data from `disk`. I think we can refactor some of these utility functions to work a bit better with composition. This may be a good pairing opportunity